enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,305 @@
+
+#### DHCP
+The Power VS environment doesn't currently have a way of handling DHCP within the environment. We've asked their team to
+look into how they can provide this, since static IPs are currently not supported for IPI installs.
+
+The team has proposed a phased approach to DHCP support. The first stage would involve providing DHCP services by
+deploying a Centos (or possibly RHEL if a subscription model can be sorted out) VM to the Power VS service. This would
+not be hidden from the user's account, but they would not be charged for its use. The goal would be for this to be
+available for any application that needs DHCP, not just OpenShift.
+
+#### Load Balancing
+When it came to load balancer, the best option we came up with was to use the IBM Cloud [VPC
+Loadbalancer][vpc-loadbalancer-website] using the Direct Link, since it allows us to provide an application load
+balancer to the cluster that can talk directly to the private network created with the [IBM Cloud DNS
+Services][ibm-cloud-dns-website]. It should be noted that the VPC Loadbalancer comes at an additional cost to the user;
+however, the Direct Link service that provides high speed connection directly to the cluster's private network is
+included with a subscriptions to Power VS.
+
+#### Separate IBM Providers for VPC vs Power VS
+One thing that has been asked and discussed is whether it is possible to create a single abstract IBM provider that
+works with IBM VPC and Power VS. Unfortunately, there are very few similiarities between the target environments outside
+of authentication. The APIs, parameters, and image management are all completely different, so we believe the cleanest
+implementation would be a separate provider.
+
+### Risks and Mitigations
+
+The Multi-Arch team has already mitigated much of the risks involved with coordinating a new ppc64le platform. We have
+existing engagements with many of the teams that will would be working with to add this platform. For instance, we
+already have a hardware environment capable of buiding the OVA images, a dedicated CI team to build out and analyze the
+development focused testing, and the underlying content release content we'd need build for ppc64le by the ART team.
+
+## Design Details
+![alt Key Components and Architecture](ibm-cloud-for-power-vs-ipi.png)
+
+The Power Systems Virtual Servers (Power VS) are separated from the rest of the IBM Cloud servers with separate networks
+and direct-attached storage. The internal networks are fenced but offer connectivity options to IBM Cloud infrastructure
+or on-premises environments. Both public and private network is supported for instances (VMs). For public networks only
+limited ports are opened for inbound access and this is not configurable.
+
+Basic Walkthrough:
+1. Create a paid IBM Cloud Account (PowerVS - or any infrastructure service, is not available for free accounts).
+2. Create Power VS service in specific region
```

Comment:
this step is done by the user, but I think we should note that all steps onward are done by either TF, bootstrapping, or in-cluster mechanisms such as the MAPI.

Reply:
Another user pre-req we'll need to add is to create a DNS domain in CIS (mentioned above)